azure-devops/projects/io-backend-projects/io-functions-publiceventdispatcher.tf

```diff
@@ -0,0 +1,177 @@
+variable "io-functions-publiceventdispatcher" {
+  default = {
+    repository = {
+      organization   = "pagopa"
+      name           = "io-functions-publiceventdispatcher"
+      branch_name    = "master"
+      pipelines_path = ".devops"
+    }
+    pipeline = {
+      cache_version_id               = "v3"
+      production_resource_group_name = "io-p-rg-pblevtdispatcher"
+      production_app_name            = "io-p-fn3-pblevtdispatcher"
```

Comment:
With new io-infra functions name starts with `io-p-fn` prefix.

The 3 in fn3 was a workaround to upgrade function runtime version from 2 to 3. Now we know that we can upgrade only the staging slot and swap to do the upgrade.

Reply:
done in https://github.com/pagopa/gitops/pull/152/commits/654b9e5ba7f5c3ba47f140bc419731e9fef3150c